packages/api/package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "develop": "concurrently \"yarn generate -w\" \"tsdx watch\"",
     "build": "graphql-codegen --config codegen.yml && tsdx build",
+    "local": "tsdx watch --entry ./local/server.ts --target node --onSuccess \"node ./dist\"",
```

Comment:
I was considering replacing the `develop` script by this... and make current one a variation, like `develop:no-server` or something like that... what do you think?

Reply:
I could see that making sense!